thinkhazard/views/api.py

```diff
@@ -24,11 +24,14 @@
 
 from ..models import DBSession, HazardCategory, HazardType, HazardLevel
 
+from ..analytics import GoogleAnalytics
+
 
 @view_config(route_name="api_hazardcategory", renderer="json")
 def api_hazardcategory(request):
     hazard_type = request.matchdict["hazard_type"]
     hazard_level = request.matchdict["hazard_level"]
+    GoogleAnalytics().hit(request.path)
```

Comment:
I think GA request will be blocking that way.
You may take a look at https://github.com/ross/requests-futures to have non-blocking requests.
(or maybe @arnaud-morvan will have other options)

Reply:
Ok thanks. Do you mean blocking in the way that the code is waiting for the response or that GA does not take the request into account? I still have the impression that the code is executed, as the API continues to serve the response when the hit is sent to GA.

Reply:
Yes, I think it will wait for GA answer before returning API response. So if GA is down API too (unlikely). But should slow down API response a bit.

Reply:
Thanks I used requests-futures to make request async after trying with asyncio